html/components/alerts.js

```diff
@@ -1,7 +1,7 @@
 import getElements from '../utilities/getElements';
 
 const dismissAlert = (alert) => {
-  alert.classList.add('sprk-u-Display--none');
+  alert.classList.add('sprk-c-Alert--is-hidden');
```

Comment:
Added new is-hidden classes for most components. We got lucky with angular and react in most cases in that they use either ngIf or conditionals for rendering.